app/javascript/legacy_react/src/lib/utils.ts

```diff
@@ -15,3 +15,7 @@ export function castToUndefinedIfBlank(i:null|undefined|string) :
     string | undefined {
   return isBlank(i) ? undefined : i;
 }
+
+export function removeChar(i:null|undefined|string, charArray:null|undefined|string) {
+  return i.replace(new RegExp('[' + charArray + ']+', 'g'), '')
+}
```

Comment:
This is good work. Would you be able to create some Jest unit tests for this? I think it'd help verify that this works in all of the situations we expect it to.

Reply:
![image](https://user-images.githubusercontent.com/49957412/180865940-1c4338b9-5f4c-42f8-8266-c37a3ff46906.png)
@wwahammy Do you know what could be cousing this error? We are trying to run tests for the the removeChar function but for some reason when we run the yarn jest we get this error.

Here is how we are testing:
![image](https://user-images.githubusercontent.com/49957412/180866359-0a550e8d-216b-4c65-8043-d27ae6583faa.png)

Reply:
When we run the test only in this file it passes:
![image](https://user-images.githubusercontent.com/49957412/180866994-d18e555f-64d7-4e63-bdcc-1209474cfb86.png)